lib/assets/javascripts/turbograft/turbohead.coffee

```diff
@@ -14,6 +22,10 @@ class window.TurboHead
       .filter(attributeMatches('nodeName', 'LINK'))
       .filter(noAttributeMatchesIn('href', @activeAssets))
 
+  @reset: ->
```

Comment:
Maybe we have the domain model wrong?

Maybe a `Turbohead` instance wants to be where the cache lives, created at time of construction, and then a `process` method on a turbohead object returns some kind of `headAssets` object which we use for `update`. Then we'd have `Turbolinks` create it's `Turbohead`, and in tests you could just create a new one when you need to.